iceoryx_hoofs/include/iceoryx_hoofs/cxx/attributes.hpp

```diff
@@ -41,8 +45,15 @@ inline void IOX_DISCARD_RESULT_IMPL(T&&) noexcept
 ///     uint32_t foo();
 ///     IOX_DISCARD_RESULT(foo()); // suppress compiler warning for unused return value
 /// @endcode
+// AXIVION Next Construct AutosarC++19_03-A16.0.1 : Aliasing of fully qualified templated function.
+//                                                  Improves readability. No risks apparent.
 #define IOX_DISCARD_RESULT(expr) ::iox::cxx::internal::IOX_DISCARD_RESULT_IMPL(expr)
 
+// AXIVION DISABLE STYLE AutosarC++19_03-A16.0.1 : Standardizes use of compiler attributes for all
```

Comment:
I have to say this is exactly how a justification should look like. I really love it!